torchvision/transforms/transforms.py

```diff
@@ -164,7 +168,16 @@ def __call__(self, img):
         return F.resize(img, self.size, self.interpolation)
 
     def __repr__(self):
-        return self.__class__.__name__ + '(size={0})'.format(self.size)
+        interpolate_str = 'PIL.Image.'
```

Comment:
I think it might be useful to have those in a private dict somewhere in this file, as it can be used by other methods as well (at least 3 with https://github.com/pytorch/vision/pull/411)
Something like
```python
_pil_interpolation_to_name_map = {
    Image.NEAREST: 'PIL.Image.NEAREST',
    Image.BILINEAR: 'PIL.Image.BILINEAR',
    ...
}
```

What do you think?

Reply:
I thought of adding something like this, but given the number of transforms that _actually_ require it, I skipped it. I could add it, if you would like me to.

Reply:
Yeah, let's add it and then #411 can use it as well.